### ISSUE TITLE
[defect]: set `lifetime` for aws-nlb site

### ISSUE THREAD

**Issue by joel-aws** [FreeRADIUS/freeradius-server#5300]
### What type of defect/bug is this?

Unexpected behaviour (obvious or verified by project member)

### How can the issue be reproduced?

* Enable the [aws-nlb](https://github.com/FreeRADIUS/freeradius-server/blob/v3.2.x/raddb/sites-available/aws-nlb) site.
* Open >16 TCP connections to the server `for i in {1..17}; do nc 127.0.0.1 8080; done`
* You will get `Failed inserting TCP socket into parent list.` errors and new TCP connections will be blocked until the idle_time of 30s is hit, which is usually the Health Check default time.

I believe that by setting the `lifetime` in the example `aws-nlb` file to `2` or so, that will suffice in clearing the stack and not letting hanging health-checks clog it up.

### Log output from the FreeRADIUS daemon

```shell
Reached maximum lifetime on socket status from client (192.168.65.1, 47195) -> (*, 8080, virtual-server=aws-nlb)
 ... shutting down socket status from client (192.168.65.1, 47195) -> (*, 8080, virtual-server=aws-nlb)
Ready to process requests
 ... new connection request on TCP socket
Failed inserting TCP socket into parent list.
```

**Comment by alandekok** [comment]
Yes, there should be a flag there saying "close socket immediately", as there's no reason to keep it around.
